app/services/BarGraphService.service.js

```diff
@@ -0,0 +1,76 @@
+(function() {
+  'use strict';
+
+  class BarGraphService {

```

Comment:
byloby fajnie, gdybys podzielila kod na kawalki i ponazywala je odpowiednio. bo za x czasu nie bedziesz wiedziec, co robia poszczegolne fragmenty kodu. a dla osoby wchodzacej do tego kodu bezposrednio po Tobie bylaby to masakra.